packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -170,7 +170,7 @@ class ServiceExtensionSelector {
   ServiceExtensionSelector(this.extensionDescription) {
     selector = PSelect()
       ..small()
-      ..clazz('toggle-platform')
+      ..clazz('on-page-selector')
```

Comment:
nit: what does `on-page-selector` mean? The term selector is a bit overused because of css query selectors so it might be good to use something different. Similarly, does "on-page" mean "in the flutter application"? 
"flutter-setting"?

Reply:
There is another selector (isolate selector in the page footer) that doesn't need this style, so I was trying to differentiate between that selector and selectors that are "on the page". Changed it to "button-bar-selector" because the styling is to match the selector to it's neighboring buttons.